### DISCUSSION TITLE
A large error of fitting a Lennard-Jones potential solid

### DISCUSSION THREAD

**Discussion by phys-chem** [deepmodeling/deepmd-kit#2881] [Q&A]
I am doing a test by using Lennard-Jones to sample some structures. Then, I fit these structures with deepmd for the test, but the RMSE is very high.  I try to adjust my parameters and still cannot solve it. Could you give me some suggestions? The attachment are my data and input,json Thank you very much!

Here is the fitting error:
(Units are reduce units LJ in LAMMPS, not eV)
DEEPMD INFO    # number of test data : 61 
DEEPMD INFO    Energy MAE         : 2.294327e-01 eV
DEEPMD INFO    Energy RMSE        : 4.881255e-01 eV
DEEPMD INFO    Energy MAE/Natoms  : 1.911939e-02 eV
DEEPMD INFO    Energy RMSE/Natoms : 4.067713e-02 eV
DEEPMD INFO    Force  MAE         : 2.293103e-02 eV/A
DEEPMD INFO    Force  RMSE        : 7.531464e-02 eV/A
DEEPMD INFO    Virial MAE         : 2.434960e+00 eV
DEEPMD INFO    Virial RMSE        : 7.304477e+00 eV
DEEPMD INFO    Virial MAE/Natoms  : 2.029133e-01 eV
DEEPMD INFO    Virial RMSE/Natoms : 6.087064e-01 eV

[fit.zip](https://github.com/deepmodeling/deepmd-kit/files/12765509/fit.zip)

**Comment by njzjz** [comment]
What's the conversion between LJ length and Angstrom? I see you set rcut to 2.

**Reply by phys-chem** [reply]
Thank you for the reply! The LJ cutoff is two, and I use the LJ unit. I just test the fitting and it is not a real system.
